### DISCUSSION TITLE
swiper无法禁止滑动

### DISCUSSION THREAD

**Discussion by Qiangggggg** [NervJS/taro#13173] [Q & A] [answered]
### 这个特性解决了什么问题？
当不满足某个条件时，禁止滑动

### 这个 API 长什么样？
希望和原生保持一致

**Comment by TheKonka** [comment]
微信小程序吗？原生是怎么禁止滑动的？

**Comment by Qiangggggg** [comment]
`<swiper-item  catchtouchmove='onTouchMove'></swiper-item>`

`onTouchMove() {
  return false
}
`

**Comment by molunhui** [comment, marked as answer]
可以在内部加一层view包裹，然后VIew 增加catchMove

**Reply by wqq20170106** [reply]
华为元服务如果在<swiper-item>外层加view会导致无法自动滚动
